clients/java/build.gradle

```diff
@@ -0,0 +1,35 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * This generated file contains a sample Java Library project to get you started.
+ * For more details take a look at the Java Libraries chapter in the Gradle
+ * user guide available at https://docs.gradle.org/4.8/userguide/java_library_plugin.html
+ */
+
+plugins {
+    // Apply the java-library plugin to add support for Java Library
+    id 'java-library'
+}
+
+dependencies {
+    // This dependency is exported to consumers, that is to say found on their compile classpath.
+    api 'org.apache.commons:commons-math3:3.6.1'
+
+    // This dependency is used internally, and not exposed to consumers on their own compile classpath.
+    implementation 'com.google.guava:guava:23.0'
+
+    // Use JUnit test framework
+    testImplementation 'junit:junit:4.12'
+
+    //Java JSON Library 
+    implementation 'com.google.code.gson:gson:2.8.5'
```

Comment:
Are these dependencies required to build or just sample dependencies that come with the template? It looks like the latter to me (`commons-math`, `guava`, `gson` don't seem to be used)

Reply:
I added this one and forgot to remove it. The rest came out of the box. Commented them out and it still builds successfully 🤷‍♂️

Reply:
Except JUnit of course